client/state/README.md

```diff
@@ -17,6 +17,19 @@ const store = createReduxStore();
 
 All the application information and data in Calypso should go through this data flow. When you are creating a new module with new data requirements, you should add them to this global store.
 
+Alternatively, you can do the following to add a reducer at run-time within the application.
+Also, make sure you remove the reducer if you code is no longer active within the system.
```

Comment:
- `s/you code/your code/` :) (also, I'd do `s/run-time/runtime/`)

- Though I personally like explicit line breaks, we usually keep one line per paragraph in Markdown documents.